CMakeLists.txt

```diff
@@ -151,7 +151,7 @@ endif()
 
 cmake_dependent_option(
     CMAKE_INTERPROCEDURAL_OPTIMIZATION "Enable interprocedural optimization?"
-    TRUE "SCP_IPO_SUPPORTED" FALSE)
+    TRUE "SCP_IPO_SUPPORTED;SCP_ENABLE_IPO" FALSE)
```

Comment:
This check was done in lines 146 to line 149. Why do we need to do it again here?

Reply:
line 139 check_ipo_supported(RESULT SCP_IPO_SUPPORTED) can have already set SCP_IPO_SUPPORTED.
but the product doesn't want to use it